app/views/admin/stores/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1><%= @store.name %></h1>
+
+<%= link_to t('store.all'), admin_stores_path %>
+<%= link_to t('store.edit'), edit_admin_store_path(@store) %>
+<%= link_to t('store.destroy'), admin_store_path(@store), data: {
+    turbo_method: :delete,
+    turbo_confirm: t('general.confirm_destroy')
+} %>
+
+</br></br>
+<b><%= t('activerecord.attributes.address.cep') %>:</b> <%= @store.address.cep %> </br>
```

Comment:
aqui vc pode usar o delegate. tipo la no model de store vc pode colocar algo como: `delegate :cep, :state, etc, to: :address`, ai vc pode acessar `@store.cep`. Vc pode usar delegate para qualquer classe, nao apenas nos models

Reply:
https://apidock.com/rails/Module/delegate